vote.py

```diff
@@ -0,0 +1,172 @@
+from brutal.core.plugin import BotPlugin, cmd
+
+
+class Vote(BotPlugin):
+    def setup(self, *args, **kwargs):
+        self.reset()
+
+    def reset(self):
+        """Reset all voting variables"""
+        self.voting = False
+        self.voting_q = False
+        self.yeses = 0
+        self.nos = 0
+        self.voted = {}
+
+    def judgment(self, event=None, penalty=''):
```

Comment:
I'd probably prefer the british spelling (judgement) but it's a matter of taste.